mod_reforged/hooks/config/strings.nut

```diff
@@ -844,12 +844,13 @@ foreach (vanillaDesc in vanillaDescriptions)
  	}),
  	RF_Bloodlust = ::UPD.getDescription({
  		Fluff = "When surrounded by carnage, you feel revitalized and right at home!",
- 		Requirement = "Melee Attack",
+ 		Requirement = "Cleaver",
  		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"During your [turn|Concept.Turn], every successful attack reduces current [Fatigue|Concept.Fatigue] by " + ::MSU.Text.colorGreen("5%") + " per stack of [Bleeding|Skill+bleeding_effect] on the target and increases [Fatigue Recovery|Concept.FatigueRecovery] by " + ::MSU.Text.colorGreen("+1") + " for one [turn|Concept.Turn] per stack of [Bleeding|Skill+bleeding_effect] on the target.",
-				"[Bleeding|Skill+bleeding_effect] inflicted by the attack, or killing a target, also counts towards the bonus."
+				"After inflicting a fatality, gain " + ::MSU.Text.colorGreen("2") + " stacks of Bloodlust, up to a maximum of 2 stacks.",
+				"For each stack [Resolve|Concept.Bravery] and [Initiative|Concept.Initiative] are increased by " + ::MSU.Text.colorGreen("25%") + " and skills build up " + ::MSU.Text.colorGreen("25%") + " less [Fatigue|Concept.Fatigue], stacking [additively|Concept.StackAdditively].",
+				"You lose " + ::MSU.Text.colorRed("1") + " stack every [turn|Concept.Turn] but a fatality will reset it to " + ::MSU.Text.colorGreen("2") + "."
```

Comment:
The second part of this bullet point is no longer needed as it is now fully covered by the first bullet point.